sdk/src/trace/span.cc

```diff
@@ -78,8 +78,9 @@ Span::Span(std::shared_ptr<Tracer> &&tracer,
   recordable_->SetName(name);
 
   trace_api::TraceId trace_id;
-  trace_api::SpanId span_id = GenerateRandomSpanId();
-  bool is_parent_span_valid = false;
+  trace_api::SpanId span_id         = GenerateRandomSpanId();
+  trace_api::TraceFlags trace_flags = parent_span_context.trace_flags();
```

Comment:
I was thinking about propagating `trace_flags only for valid `SpanContext`, but since we could create a invalid `SpanContext` with non-zero `trace_flags`, so I choose to do propagation even for invalid span. Let me know if you prefer the other case.